#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_default_action.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Serveral changes/additons to default.action.

@DPATCH@
diff -urNad privoxy~/default.action.master privoxy/default.action.master
--- privoxy~/default.action.master
+++ privoxy/default.action.master
@@ -1839,6 +1839,11 @@
 {+filter{tiny-textforms} +prevent-compression}
 .sourceforge.net/tracker
 
+{-filter{ie-exploits}}
+#MASTER# REMARKS: Allow %01 and %02 in URLs as they are used in Amazon's 
+#MASTER# REMARKS: multi-page results see http://bugs.debian.org/243245
+.amazon.de
+
 {+downgrade-http-version}
 #MASTER# COMMENTS: This section not checked 10/17/06 HB
 #MASTER# REMARKS: This is work-around for CUPS http configuration.
